scripts_src/arroyo/aibroc.ssl

```diff
@@ -34,27 +34,38 @@
 #include "../headers/command.h"
 
 procedure start;
+procedure timed_event_p_proc;
 procedure pickup_p_proc;
 procedure destroy_p_proc;
-procedure drop_p_proc;
+//procedure drop_p_proc;
```

Comment:
Don't need dead code, here or anywhere else.

Reply:
Should I delete killap`s commented lines too then?
```
   // comment out by killap - acdog.int now makes the dog go invisible, rather than killing him
   //kill_critter_type(PID_DOG,KILL_DONT_LEAVE_BODY);
```

Reply:
No need to go hunting for them specifically, but if such lines are in proximity of changes being made, then yes, cleaning them up is fine. I do that all the time.